### ISSUE TITLE
docs: `coder whoami`

### ISSUE THREAD

**Issue by sharkymark** [coder/coder#14024]
The [docs](https://coder.com/docs/cli/whoami) reference a CLI command `coder whoami` but it does not resolve for me.

`coder --help` does not show it either.

**Comment by matifali** [comment]
It got merged in https://github.com/coder/coder/pull/13814 3 weeks ago and hasn't been released yet. 
It will be the part of next Coder releases. 
@stirby we should default our docs to release branch/tag instead of `main` to avoid such confusions in future. 

We already have the necessary tooling and current release docs are available at https://coder.com/docs/@v2.13.2

Or if we want to point docs to latest(stable) it's https://coder.com/docs/@v2
12.5 
Stable version can be [easily inquired](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release) from GitHub with

```shell
gh api /repos/coder/coder/releases/latest | jq -r '.tag_name' 
```

**Comment by matifali** [comment]
It got merged in #13966 13814 3 weeks ago and hasn't been released yet. It will be the part of next Coder release.
